### ISSUE TITLE
Roland VSTi TR-808 stop working by pushing EDIT button.

### ISSUE THREAD

**Issue by Inqb8tr-jp** [falkTX/Carla#1722]
Maybe it's not Carla related but I haven't TR-808 VSTi work on Carla under Manjaro Linux.

- both carla and carla-git (Also Ardour7.2 has almost same problem)
- When I pushing the EDIT button, the TR-808 freeze.
- Also a new window can't be visible when I pushing the EDIT button like [https://twitter.com/Inqb8tr_jp/status/1606888275947687936/photo/1](https://twitter.com/Inqb8tr_jp/status/1606888275947687936/photo/1)

TR-808 works only when Roland Cloud Manager is working, probably for licensing.
I'm a subscriber of PRO plan for Roland Cloud Manager for a month.

```======= Starting engine =======
======= Engine started ========
Carla engine started, details:
  Driver name:  JACK
  Sample rate:  48000
  Process mode: Multi client
18:03:58 [TR-808(VST2 64bit)-1349Icey] Initializing yabridge version 5.0.2
18:03:58 [TR-808(VST2 64bit)-1349Icey] library:       '/usr/lib/libyabridge-vst2.so'
18:03:58 [TR-808(VST2 64bit)-1349Icey] host:          '/usr/bin/yabridge-host.exe'
18:03:58 [TR-808(VST2 64bit)-1349Icey] plugin:        '/home/aki/.wine/drive_c/Program Files/Vstplugins/Roland/TR-808/TR-808(VST2 64bit).dll'
18:03:58 [TR-808(VST2 64bit)-1349Icey] plugin type:   'VST2'
18:03:58 [TR-808(VST2 64bit)-1349Icey] realtime:      'yes'
18:03:58 [TR-808(VST2 64bit)-1349Icey] sockets:       '/run/user/1000/yabridge-TR-808(VST2 64bit)-1349Icey'
18:03:58 [TR-808(VST2 64bit)-1349Icey] wine prefix:   '/home/aki/.wine'
18:03:58 [TR-808(VST2 64bit)-1349Icey] wine version:  '7.22'
18:03:58 [TR-808(VST2 64bit)-1349Icey] 
18:03:58 [TR-808(VST2 64bit)-1349Icey] config from:   '<defaults>'
18:03:58 [TR-808(VST2 64bit)-1349Icey] hosting mode:  'individually, 64-bit'
18:03:58 [TR-808(VST2 64bit)-1349Icey] other options: '<none>'
18:03:58 [TR-808(VST2 64bit)-1349Icey] 
18:03:58 [TR-808(VST2 64bit)-1349Icey] Enabled features:
18:03:58 [TR-808(VST2 64bit)-1349Icey] - bitbridge support
18:03:58 [TR-808(VST2 64bit)-1349Icey] - CLAP support
18:03:58 [TR-808(VST2 64bit)-1349Icey] - VST3 support
18:03:58 [TR-808(VST2 64bit)-1349Icey] 
18:03:58 [TR-808(VST2 64bit)-1349Icey] [Wine STDERR] Initializing yabridge host version 5.0.2
18:03:58 [TR-808(VST2 64bit)-1349Icey] [Wine STDERR] Preparing to load VST2 plugin at '/home/aki/.wine/drive_c/Program Files/Vstplugins/Roland/TR-808/TR-808(VST2 64bit).dll'
18:03:59 [TR-808(VST2 64bit)-1349Icey] [Wine STDERR] 0600:fixme:toolhelp:CreateToolhelp32Snapshot Unimplemented: heap list snapshot
18:03:59 [TR-808(VST2 64bit)-1349Icey] [Wine STDERR] 0600:fixme:toolhelp:Heap32ListFirst : stub
18:03:59 [TR-808(VST2 64bit)-1349Icey] [Wine STDERR] 0600:fixme:ntdll:NtQuerySystemInformation info_class SYSTEM_PERFORMANCE_INFORMATION
18:03:59 [TR-808(VST2 64bit)-1349Icey] [Wine STDERR] 0604:fixme:kernelbase:AppPolicyGetThreadInitializationType FFFFFFFFFFFFFFFA, 00000000021FFE10
18:03:59 [TR-808(VST2 64bit)-1349Icey] [Wine STDERR] 0610:fixme:file:ReplaceFileW Ignoring flags 6
18:03:59 [TR-808(VST2 64bit)-1349Icey] [Wine STDERR] 060c:fixme:file:ReplaceFileW Ignoring flags 6
18:03:59 [TR-808(VST2 64bit)-1349Icey] [Wine STDERR] Finished initializing '/home/aki/.wine/drive_c/Program Files/Vstplugins/Roland/TR-808/TR-808(VST2 64bit).dll'
18:04:01 [TR-808(VST2 64bit)-1349Icey] [Wine STDERR] 0600:fixme:imm:ImeSetActiveContext (00000000000D00B8, 1): stub
18:04:01 [TR-808(VST2 64bit)-1349Icey] [Wine STDERR] 0600:fixme:imm:ImmReleaseContext (00000000000300AE, 00000000000D00B8): stub
18:04:39 [TR-808(VST2 64bit)-1349Icey] [Wine STDERR] 0600:fixme:msg:pack_message msg 14 (WM_ERASEBKGND) not supported yet
```

**Comment by Inqb8tr-jp** [comment]
As I follow the yarbridge's explanation of configuration, by adding "editor_xembed=true" on yabridge.toml, I can open edit button like this.
[https://twitter.com/Inqb8tr_jp/status/1616824069927886850](https://twitter.com/Inqb8tr_jp/status/1616824069927886850)

But still the bottom of the window is cut off.
I tried spread down the window but can't change the window size.
I tried all available option of yabridge but no good.
